modules/observability-monitoring-the-edge.adoc

```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 //
-// * edge_computing/day_2_core_cnf_clusters/observability/telco-observability.adoc
+// * edge_computing/day_2_core_cnf_clusters/observability/observability.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="telco-observability-monitoring-the-edge_{context}"]
+[id="observability-monitoring-the-edge_{context}"]
 
 = Monitoring the edge
 
-{sno-caps} at the edge keeps the footprint of the platform components to a minimum. 
-The following procedure is an example of how you can configure a {sno} node with a small monitoring footprint.
+OpenShift clusters at the edge should keep the footprint of the platform components to a minimum. 
+The following procedure is an example of how you can configure a {sno} or edge node with a small monitoring footprint.
```

Comment:
Same comment as above, in this section, you could refer to "nodes at the far edge of the network", rather than just "the edge" or "edge node"